src/Source/C3DTilesSource.js

```diff
@@ -1,9 +1,21 @@
 import Source from 'Source/Source';
 import Fetcher from 'Provider/Fetcher';
 
+/**
+ * @classdesc
+ * An object defining the source connection to a 3DTiles dataset from a web server.
+ *
+ * @extends Source
+ *
+ * @property {boolean} isC3DTilesSource - Used to checkout whether this source is a isC3DTilesSource. Default is
+ * true. You should not change this, as it is used internally for optimisation.
+ * @property {string} url - The URL of the tileset json.
+ * @property {string} baseUrl - The base URL to access tiles.
+ */
 class C3DTilesSource extends Source {
     constructor(source) {
```

Comment:
Here again, could we add doc on the `constructor`'s parameters ?

Reply:
yes, done too.